yb-voyager/cmd/exportData.go

```diff
@@ -326,6 +333,30 @@ func writeDataFileDescriptor(exportDir string, status *dbzm.ExportStatus) error
 	return nil
 }
 
+func createResumeSequencesFile() {
+	status, err := dbzm.ReadExportStatus(filepath.Join(exportDir, "data", "export_status.json"))
+	if err != nil {
+		utils.ErrExit("Failed to read export status during creating resume sequence file: %v", err)
+	}
+
+	var sqlStmts []string
+	for sequenceName, lastValue := range status.Sequences {
+		if lastValue == 0 {
+			continue
+		}
+		sqlStmt := fmt.Sprintf("SELECT pg_catalog.setval('%s', %d, true);\n", sequenceName, lastValue)
+		sqlStmts = append(sqlStmts, sqlStmt)
+	}
+
+	if len(sqlStmts) > 0 {
```

Comment:
No file is created if there are no sequences?

Reply:
Yes